src/main/java/com/ss/ita/rozetka/ui/pages/ProductPage.java

```diff
@@ -1,5 +1,13 @@
 package com.ss.ita.rozetka.ui.pages;
 
+import com.ss.ita.rozetka.ui.Modals.BasketModal;
+
+import static com.codeborne.selenide.Selenide.$x;
+
 public class ProductPage extends Header {
-    //for writing methods in close future
+
+    public BasketModal addProductToBasket() {
```

Comment:
what product?

Reply:
we have only one product on product page. Any product that U choose before.

Reply:
maybe rename like addChosenProductToBasket?